lib/tldr.js

```diff
@@ -24,8 +24,8 @@ exports.listAll = (singleColumn) => {
   });
 };
 
-exports.get = (commands) => {
-  printBestPage(commands.join('-'));
+exports.get = (commands, options) => {
+  printBestPage(commands.join('-'), options);
```

Comment:
Please add this line inside printBestPage function to convert to empty dictionary if `options` is not passed.

```
options = options || {};
```

to fix the travis error.